px-central/values.yaml

```diff
@@ -115,38 +116,23 @@ images:
   pxcentralFrontendImage:
     registry: docker.io
     repo: portworx
-    imageName: pxcentral-onprem-ui-frontend
-    tag: 1.2.2
+    imageName: pxcentral-onprem-ui-frontend-private
+    tag: 1.3.0-dev
   pxcentralBackendImage:
     registry: docker.io
     repo: portworx
-    imageName: pxcentral-onprem-ui-backend
-    tag: 1.2.2
+    imageName: pxcentral-onprem-ui-backend-private
+    tag: 1.3.0-dev
   pxcentralMiddlewareImage:
     registry: docker.io
     repo: portworx
-    imageName: pxcentral-onprem-ui-lhbackend
-    tag: 1.2.2
-  pxBackupImage:
-    registry: docker.io
-    repo: portworx
-    imageName: px-backup-base
-    tag: master
+    imageName: pxcentral-onprem-ui-lhbackend-private
+    tag: 1.3.0-dev
```

Comment:
All the UI images need to point to master and not 1.3.0-dev

Reply:
Done.